File: src/testgr/wxg.go
package main

import "fmt"

func main() {
	r := foo(4)
	fmt.Println(r)
}

func foo(n int) int {
	// 输入一个正整数,输出正整数平方根
	var y int = 0
	if n < 0 {
		return -1
	}
	if n == 0 {
		return 0
	}

	if y*y < n {
		y++
	} else {
		return y
	}
}

func bar(li []int) int {
	// 输入一个数组,输出数组中两次出现的值
	var a int = 0
	var b int = 0
	for i := 0; i < len(li); i++ {
		if li[i] == a {
			b = a
			a = li[i]
		} else if li[i] == b {
			b = li[i]
		}
	}
	return b
}

File: src/testgr/wxg.py

def bar(n):
    x  = 0
    if n  < 0:
        return  -1 
    if n == 0:
        return 0
    while True:
        if x*x < n:
            x += 1
        else:
            return x
        
def foo(li):
    # lib = set(li)
    tmp = dict()
    li_b = []
    for i in li:
        if i in tmp:
            tmp[i] += 1
            li_b.append(i)
        else:
            tmp[i] = 1
    
    print(li_b)

if __name__ == "__main__":
    li = [1, 2, 3, 3]
    foo(li)
    print(bar(4))
